sequencer/src/api/options.rs

```diff
@@ -168,6 +168,29 @@ impl Options {
                 "API server",
                 app.serve(format!("0.0.0.0:{}", self.http.port), bind_version),
             );
+
+            // It runs to different port and app because State and Extensible Data source needs to support required
```

Comment:
Do we not initialize this in the previous case or am I missing it (`if let Some(query_opt) = self.query.take() {`)?